eng/Versions.props

```diff
@@ -9,7 +9,7 @@
     <AspNetCoreMajorVersion>3</AspNetCoreMajorVersion>
     <AspNetCoreMinorVersion>0</AspNetCoreMinorVersion>
     <AspNetCorePatchVersion>0</AspNetCorePatchVersion>
-    <PreReleasePreviewNumber>8</PreReleasePreviewNumber>
+    <PreReleasePreviewNumber>9</PreReleasePreviewNumber>
```

Comment:
Should this PR actually target 'release/3.0'?

Reply:
Yes, totally. I'll  fix it when I get back to this.